src/System.Web.Compatibility/System.Web.manual.cs

```diff
@@ -13,16 +16,43 @@ public partial class HttpContext
         public static HttpContext Current
         {
             get => null;
-            set { throw new System.PlatformNotSupportedException(Strings.PlatformNotSupportedSystemWeb); }
+            set => throw new PlatformNotSupportedException(Strings.PlatformNotSupportedSystemWeb);
         }
+
+        public IDictionary Items
+        {
+            get => null;
+        }
+
+        public IPrincipal User
+        {
+            get => null;
+            set => throw new PlatformNotSupportedException(Strings.PlatformNotSupportedSystemWeb);
+        }
+
+        public HttpContext(HttpRequest request, HttpResponse response) { }
```

Comment:
If you want to define the constructor you'll need to suppress it in the exclusions file.  The syntax to add for these 3 constructors is as follows:
```
M:System.Web.HttpContext.#ctor(System.Web.HttpRequest,System.Web.HttpResponse)
M:System.Web.HttpRequest.#ctor(System.String,System.String,System.String)
M:System.Web.HttpResponse.#ctor(System.IO.TextWriter)
```

Reply:
I see you had these, but had spaces.  The docID format is rather rigid here.  Let me fix this up for you.